server/src/main/java/io/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -233,6 +233,35 @@ public boolean isLeader()
 
   public CountingMap<String> getSegmentAvailability()
   {
+    return getSegmentStats(
+        new Function<DataSegment, Integer>()
+        {
+          @Override
+          public Integer apply(DataSegment input)
+          {
+            int available = (segmentReplicantLookup.getTotalReplicants(input.getIdentifier()) == 0) ? 0 : 1;
```

Comment:
Do you mean 
`return segmentReplicantLookup.getTotalReplicants(input.getIdentifier()) == 0 ? 1 : 0`
? I cannot understand.